pylbm/__init__.py

```diff
@@ -7,47 +7,52 @@
 """
 A flexible Python package for lattice Boltzmann method.
 """
+
 import logging
 from colorlog import ColoredFormatter
 import mpi4py.MPI as mpi
-
 from colorama import init
-init()
```

Comment:
you moved the init call. Could you put it again at this place?

Reply:
Yes but it causes a warning when using pylint. I'm deactivating it!